jquery.xblock.js

```diff
@@ -214,7 +221,7 @@
 
                         xhr.setRequestHeader('X-Requested-With', 'XMLHttpRequest');
 
-                        if (!$this.csrfSafeMethod(settings.type) && queryDomain === lmsDomain) {
+                        if (!$this.csrfSafeMethod(settings.type)) {
```

Comment:
@e-kolpakov The one on jquery-xblock is the right one - but you seem to have reverted to that already.